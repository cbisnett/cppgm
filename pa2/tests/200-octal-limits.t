00017777777777 00017777777777 00017777777777l 00017777777777l 00017777777777L 00017777777777L 00017777777777ll 00017777777777ll 00017777777777LL 00017777777777LL 
00017777777777u 00017777777777u 00017777777777ul 00017777777777lu 00017777777777uL 00017777777777Lu 00017777777777ull 00017777777777llu 00017777777777uLL 00017777777777LLu 
00017777777777U 00017777777777U 00017777777777Ul 00017777777777lU 00017777777777UL 00017777777777LU 00017777777777Ull 00017777777777llU 00017777777777ULL 00017777777777LLU 
00017777777777_ud1 00017777777777_ud1 00017777777777_ud1l 00017777777777l_ud1 00017777777777_ud1L 00017777777777L_ud1 00017777777777_ud1ll 00017777777777ll_ud1 00017777777777_ud1LL 00017777777777LL_ud1 
017777777776 017777777776 017777777776l 017777777776l 017777777776L 017777777776L 017777777776ll 017777777776ll 017777777776LL 017777777776LL 
017777777776u 017777777776u 017777777776ul 017777777776lu 017777777776uL 017777777776Lu 017777777776ull 017777777776llu 017777777776uLL 017777777776LLu 
017777777776U 017777777776U 017777777776Ul 017777777776lU 017777777776UL 017777777776LU 017777777776Ull 017777777776llU 017777777776ULL 017777777776LLU 
017777777776_ud1 017777777776_ud1 017777777776_ud1l 017777777776l_ud1 017777777776_ud1L 017777777776L_ud1 017777777776_ud1ll 017777777776ll_ud1 017777777776_ud1LL 017777777776LL_ud1 
020000000000 020000000000 020000000000l 020000000000l 020000000000L 020000000000L 020000000000ll 020000000000ll 020000000000LL 020000000000LL 
020000000000u 020000000000u 020000000000ul 020000000000lu 020000000000uL 020000000000Lu 020000000000ull 020000000000llu 020000000000uLL 020000000000LLu 
020000000000U 020000000000U 020000000000Ul 020000000000lU 020000000000UL 020000000000LU 020000000000Ull 020000000000llU 020000000000ULL 020000000000LLU 
020000000000_ud1 020000000000_ud1 020000000000_ud1l 020000000000l_ud1 020000000000_ud1L 020000000000L_ud1 020000000000_ud1ll 020000000000ll_ud1 020000000000_ud1LL 020000000000LL_ud1 
037777777777 037777777777 037777777777l 037777777777l 037777777777L 037777777777L 037777777777ll 037777777777ll 037777777777LL 037777777777LL 
037777777777u 037777777777u 037777777777ul 037777777777lu 037777777777uL 037777777777Lu 037777777777ull 037777777777llu 037777777777uLL 037777777777LLu 
037777777777U 037777777777U 037777777777Ul 037777777777lU 037777777777UL 037777777777LU 037777777777Ull 037777777777llU 037777777777ULL 037777777777LLU 
037777777777_ud1 037777777777_ud1 037777777777_ud1l 037777777777l_ud1 037777777777_ud1L 037777777777L_ud1 037777777777_ud1ll 037777777777ll_ud1 037777777777_ud1LL 037777777777LL_ud1 
037777777776 037777777776 037777777776l 037777777776l 037777777776L 037777777776L 037777777776ll 037777777776ll 037777777776LL 037777777776LL 
037777777776u 037777777776u 037777777776ul 037777777776lu 037777777776uL 037777777776Lu 037777777776ull 037777777776llu 037777777776uLL 037777777776LLu 
037777777776U 037777777776U 037777777776Ul 037777777776lU 037777777776UL 037777777776LU 037777777776Ull 037777777776llU 037777777776ULL 037777777776LLU 
037777777776_ud1 037777777776_ud1 037777777776_ud1l 037777777776l_ud1 037777777776_ud1L 037777777776L_ud1 037777777776_ud1ll 037777777776ll_ud1 037777777776_ud1LL 037777777776LL_ud1 
040000000000 040000000000 040000000000l 040000000000l 040000000000L 040000000000L 040000000000ll 040000000000ll 040000000000LL 040000000000LL 
040000000000u 040000000000u 040000000000ul 040000000000lu 040000000000uL 040000000000Lu 040000000000ull 040000000000llu 040000000000uLL 040000000000LLu 
040000000000U 040000000000U 040000000000Ul 040000000000lU 040000000000UL 040000000000LU 040000000000Ull 040000000000llU 040000000000ULL 040000000000LLU 
040000000000_ud1 040000000000_ud1 040000000000_ud1l 040000000000l_ud1 040000000000_ud1L 040000000000L_ud1 040000000000_ud1ll 040000000000ll_ud1 040000000000_ud1LL 040000000000LL_ud1 
0000000000777777777777777777777 0000000000777777777777777777777 0000000000777777777777777777777l 0000000000777777777777777777777l 0000000000777777777777777777777L 0000000000777777777777777777777L 0000000000777777777777777777777ll 0000000000777777777777777777777ll 0000000000777777777777777777777LL 0000000000777777777777777777777LL 
0000000000777777777777777777777u 0000000000777777777777777777777u 0000000000777777777777777777777ul 0000000000777777777777777777777lu 0000000000777777777777777777777uL 0000000000777777777777777777777Lu 0000000000777777777777777777777ull 0000000000777777777777777777777llu 0000000000777777777777777777777uLL 0000000000777777777777777777777LLu 
0000000000777777777777777777777U 0000000000777777777777777777777U 0000000000777777777777777777777Ul 0000000000777777777777777777777lU 0000000000777777777777777777777UL 0000000000777777777777777777777LU 0000000000777777777777777777777Ull 0000000000777777777777777777777llU 0000000000777777777777777777777ULL 0000000000777777777777777777777LLU 
0000000000777777777777777777777_ud1 0000000000777777777777777777777_ud1 0000000000777777777777777777777_ud1l 0000000000777777777777777777777l_ud1 0000000000777777777777777777777_ud1L 0000000000777777777777777777777L_ud1 0000000000777777777777777777777_ud1ll 0000000000777777777777777777777ll_ud1 0000000000777777777777777777777_ud1LL 0000000000777777777777777777777LL_ud1 
0777777777777777777776 0777777777777777777776 0777777777777777777776l 0777777777777777777776l 0777777777777777777776L 0777777777777777777776L 0777777777777777777776ll 0777777777777777777776ll 0777777777777777777776LL 0777777777777777777776LL 
0777777777777777777776u 0777777777777777777776u 0777777777777777777776ul 0777777777777777777776lu 0777777777777777777776uL 0777777777777777777776Lu 0777777777777777777776ull 0777777777777777777776llu 0777777777777777777776uLL 0777777777777777777776LLu 
0777777777777777777776U 0777777777777777777776U 0777777777777777777776Ul 0777777777777777777776lU 0777777777777777777776UL 0777777777777777777776LU 0777777777777777777776Ull 0777777777777777777776llU 0777777777777777777776ULL 0777777777777777777776LLU 
0777777777777777777776_ud1 0777777777777777777776_ud1 0777777777777777777776_ud1l 0777777777777777777776l_ud1 0777777777777777777776_ud1L 0777777777777777777776L_ud1 0777777777777777777776_ud1ll 0777777777777777777776ll_ud1 0777777777777777777776_ud1LL 0777777777777777777776LL_ud1 
01000000000000000000000 01000000000000000000000 01000000000000000000000l 01000000000000000000000l 01000000000000000000000L 01000000000000000000000L 01000000000000000000000ll 01000000000000000000000ll 01000000000000000000000LL 01000000000000000000000LL 
01000000000000000000000u 01000000000000000000000u 01000000000000000000000ul 01000000000000000000000lu 01000000000000000000000uL 01000000000000000000000Lu 01000000000000000000000ull 01000000000000000000000llu 01000000000000000000000uLL 01000000000000000000000LLu 
01000000000000000000000U 01000000000000000000000U 01000000000000000000000Ul 01000000000000000000000lU 01000000000000000000000UL 01000000000000000000000LU 01000000000000000000000Ull 01000000000000000000000llU 01000000000000000000000ULL 01000000000000000000000LLU 
01000000000000000000000_ud1 01000000000000000000000_ud1 01000000000000000000000_ud1l 01000000000000000000000l_ud1 01000000000000000000000_ud1L 01000000000000000000000L_ud1 01000000000000000000000_ud1ll 01000000000000000000000ll_ud1 01000000000000000000000_ud1LL 01000000000000000000000LL_ud1 
00000000000000000001777777777777777777777 00000000000000000001777777777777777777777 00000000000000000001777777777777777777777l 00000000000000000001777777777777777777777l 00000000000000000001777777777777777777777L 00000000000000000001777777777777777777777L 00000000000000000001777777777777777777777ll 00000000000000000001777777777777777777777ll 00000000000000000001777777777777777777777LL 00000000000000000001777777777777777777777LL 
00000000000000000001777777777777777777777u 00000000000000000001777777777777777777777u 00000000000000000001777777777777777777777ul 00000000000000000001777777777777777777777lu 00000000000000000001777777777777777777777uL 00000000000000000001777777777777777777777Lu 00000000000000000001777777777777777777777ull 00000000000000000001777777777777777777777llu 00000000000000000001777777777777777777777uLL 00000000000000000001777777777777777777777LLu 
00000000000000000001777777777777777777777U 00000000000000000001777777777777777777777U 00000000000000000001777777777777777777777Ul 00000000000000000001777777777777777777777lU 00000000000000000001777777777777777777777UL 00000000000000000001777777777777777777777LU 00000000000000000001777777777777777777777Ull 00000000000000000001777777777777777777777llU 00000000000000000001777777777777777777777ULL 00000000000000000001777777777777777777777LLU 
00000000000000000001777777777777777777777_ud1 00000000000000000001777777777777777777777_ud1 00000000000000000001777777777777777777777_ud1l 00000000000000000001777777777777777777777l_ud1 00000000000000000001777777777777777777777_ud1L 00000000000000000001777777777777777777777L_ud1 00000000000000000001777777777777777777777_ud1ll 00000000000000000001777777777777777777777ll_ud1 00000000000000000001777777777777777777777_ud1LL 00000000000000000001777777777777777777777LL_ud1 
01777777777777777777776 01777777777777777777776 01777777777777777777776l 01777777777777777777776l 01777777777777777777776L 01777777777777777777776L 01777777777777777777776ll 01777777777777777777776ll 01777777777777777777776LL 01777777777777777777776LL 
01777777777777777777776u 01777777777777777777776u 01777777777777777777776ul 01777777777777777777776lu 01777777777777777777776uL 01777777777777777777776Lu 01777777777777777777776ull 01777777777777777777776llu 01777777777777777777776uLL 01777777777777777777776LLu 
01777777777777777777776U 01777777777777777777776U 01777777777777777777776Ul 01777777777777777777776lU 01777777777777777777776UL 01777777777777777777776LU 01777777777777777777776Ull 01777777777777777777776llU 01777777777777777777776ULL 01777777777777777777776LLU 
01777777777777777777776_ud1 01777777777777777777776_ud1 01777777777777777777776_ud1l 01777777777777777777776l_ud1 01777777777777777777776_ud1L 01777777777777777777776L_ud1 01777777777777777777776_ud1ll 01777777777777777777776ll_ud1 01777777777777777777776_ud1LL 01777777777777777777776LL_ud1 
02000000000000000000000 02000000000000000000000 02000000000000000000000l 02000000000000000000000l 02000000000000000000000L 02000000000000000000000L 02000000000000000000000ll 02000000000000000000000ll 02000000000000000000000LL 02000000000000000000000LL 
02000000000000000000000u 02000000000000000000000u 02000000000000000000000ul 02000000000000000000000lu 02000000000000000000000uL 02000000000000000000000Lu 02000000000000000000000ull 02000000000000000000000llu 02000000000000000000000uLL 02000000000000000000000LLu 
02000000000000000000000U 02000000000000000000000U 02000000000000000000000Ul 02000000000000000000000lU 02000000000000000000000UL 02000000000000000000000LU 02000000000000000000000Ull 02000000000000000000000llU 02000000000000000000000ULL 02000000000000000000000LLU 
02000000000000000000000_ud1 02000000000000000000000_ud1 02000000000000000000000_ud1l 02000000000000000000000l_ud1 02000000000000000000000_ud1L 02000000000000000000000L_ud1 02000000000000000000000_ud1ll 02000000000000000000000ll_ud1 02000000000000000000000_ud1LL 02000000000000000000000LL_ud1 
