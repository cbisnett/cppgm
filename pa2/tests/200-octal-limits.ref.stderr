ERROR: malformed number (#2): 00017777777777l_ud1
ERROR: malformed number (#2): 00017777777777L_ud1
ERROR: malformed number (#2): 00017777777777ll_ud1
ERROR: malformed number (#2): 00017777777777LL_ud1
ERROR: malformed number (#2): 017777777776l_ud1
ERROR: malformed number (#2): 017777777776L_ud1
ERROR: malformed number (#2): 017777777776ll_ud1
ERROR: malformed number (#2): 017777777776LL_ud1
ERROR: malformed number (#2): 020000000000l_ud1
ERROR: malformed number (#2): 020000000000L_ud1
ERROR: malformed number (#2): 020000000000ll_ud1
ERROR: malformed number (#2): 020000000000LL_ud1
ERROR: malformed number (#2): 037777777777l_ud1
ERROR: malformed number (#2): 037777777777L_ud1
ERROR: malformed number (#2): 037777777777ll_ud1
ERROR: malformed number (#2): 037777777777LL_ud1
ERROR: malformed number (#2): 037777777776l_ud1
ERROR: malformed number (#2): 037777777776L_ud1
ERROR: malformed number (#2): 037777777776ll_ud1
ERROR: malformed number (#2): 037777777776LL_ud1
ERROR: malformed number (#2): 040000000000l_ud1
ERROR: malformed number (#2): 040000000000L_ud1
ERROR: malformed number (#2): 040000000000ll_ud1
ERROR: malformed number (#2): 040000000000LL_ud1
ERROR: malformed number (#2): 0000000000777777777777777777777l_ud1
ERROR: malformed number (#2): 0000000000777777777777777777777L_ud1
ERROR: malformed number (#2): 0000000000777777777777777777777ll_ud1
ERROR: malformed number (#2): 0000000000777777777777777777777LL_ud1
ERROR: malformed number (#2): 0777777777777777777776l_ud1
ERROR: malformed number (#2): 0777777777777777777776L_ud1
ERROR: malformed number (#2): 0777777777777777777776ll_ud1
ERROR: malformed number (#2): 0777777777777777777776LL_ud1
ERROR: malformed number (#2): 01000000000000000000000l_ud1
ERROR: malformed number (#2): 01000000000000000000000L_ud1
ERROR: malformed number (#2): 01000000000000000000000ll_ud1
ERROR: malformed number (#2): 01000000000000000000000LL_ud1
ERROR: malformed number (#2): 00000000000000000001777777777777777777777l_ud1
ERROR: malformed number (#2): 00000000000000000001777777777777777777777L_ud1
ERROR: malformed number (#2): 00000000000000000001777777777777777777777ll_ud1
ERROR: malformed number (#2): 00000000000000000001777777777777777777777LL_ud1
ERROR: malformed number (#2): 01777777777777777777776l_ud1
ERROR: malformed number (#2): 01777777777777777777776L_ud1
ERROR: malformed number (#2): 01777777777777777777776ll_ud1
ERROR: malformed number (#2): 01777777777777777777776LL_ud1
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: octal integer literal out of range (#2): 2000000000000000000000
ERROR: malformed number (#2): 02000000000000000000000l_ud1
ERROR: malformed number (#2): 02000000000000000000000L_ud1
ERROR: malformed number (#2): 02000000000000000000000ll_ud1
ERROR: malformed number (#2): 02000000000000000000000LL_ud1
